lib/testing/src/lib/core/pages/form/widgets/attach-file-widget.page.ts

```diff
@@ -122,6 +122,15 @@ export class AttachFileWidgetPage {
         return this.removeFileOptionButton.isEnabled();
     }
 
+    async isRemoveFileOptionDisplayed(): Promise<boolean> {
+        try {
+            await this.removeFileOptionButton.isDisplayed();
```

Comment:
isDisplayed doesn't need a try-catch and if you convert this element in TestElement the TestElement Class has already all the options.  A better option is convert all the element of this page in TestElement

Reply:
I've changed the check over the element, thanks.
And about changing all the elements from this page, I think it would be better to have a different task for refactoring